src/host/outputStream.cpp

```diff
@@ -849,46 +351,21 @@ bool ConhostInternalGetSet::IsVtInputEnabled() const
     return _io.GetActiveInputBuffer()->IsInVirtualTerminalInputMode();
 }
 
-// Method Description:
-// - Updates the buffer's current text attributes depending on whether we are
-//   starting/ending a hyperlink
-// Arguments:
-// - The hyperlink URI
-// Return Value:
-// - <none>
-void ConhostInternalGetSet::AddHyperlink(const std::wstring_view uri, const std::wstring_view params) const
-{
-    auto& screenInfo = _io.GetActiveOutputBuffer();
-    auto attr = screenInfo.GetAttributes();
-    const auto id = screenInfo.GetTextBuffer().GetHyperlinkId(uri, params);
-    attr.SetHyperlinkId(id);
-    screenInfo.GetTextBuffer().SetCurrentAttributes(attr);
-    screenInfo.GetTextBuffer().AddHyperlinkToMap(uri, id);
-}
-
-void ConhostInternalGetSet::EndHyperlink() const
-{
-    auto& screenInfo = _io.GetActiveOutputBuffer();
-    auto attr = screenInfo.GetAttributes();
-    attr.SetHyperlinkId(0);
-    screenInfo.GetTextBuffer().SetCurrentAttributes(attr);
-}
-
 // Routine Description:
-// - Replaces the active soft font with the given bit pattern.
+// - Lets accessibility apps know when an area of the screen has changed.
 // Arguments:
-// - bitPattern - An array of scanlines representing all the glyphs in the font.
-// - cellSize - The cell size for an individual glyph.
-// - centeringHint - The horizontal extent that glyphs are offset from center.
-// Return Value:
+// - changedRect - the area that has changed.
+// Return value:
 // - <none>
-void ConhostInternalGetSet::UpdateSoftFont(const gsl::span<const uint16_t> bitPattern,
-                                           const SIZE cellSize,
-                                           const size_t centeringHint)
+void ConhostInternalGetSet::NotifyAccessibilityChange(const til::rect changedRect)

```

Comment:
nit: If you can, pass `til::rect` by reference (and `til::point`/`til::size` by value).
Not really important for this PR though. I'll "standardize" the way we pass these types throughout the project in the future anyways.

Reply:
OK. If I make the update discussed above, I'll do this at the same time.

Reply:
If you wonder about this: It's just because the former doesn't fit into a single register (16 bytes large) and the latter two do (8 bytes). Technically I wouldn't do such a teeny tiny micro-optimization, but MSVC has sort of a bug, where 16-byte large types are spilled on the stack for parameter passing using SSE instructions (`movdqu`/`movdqa`). Each function call is then not `2.5ns` but rather about `2.5ns + 10ns * number_of_16_byte_args`. In case of the VT parser this infamously led to a ~40% _overall_ speed up of Windows Terminal on my AMD Zen 3 CPU just by not passing 16 byte `string_views` and instead pass a C-string pointer via a register argument. So... That's why I'm a bit careful about this one thing in particular now...